modules/learn/pages/security/using-multiple-cas.adoc

```diff
@@ -8,15 +8,18 @@
 
 Prior to Couchbase Server Version 7.1, a Couchbase-Server cluster could maintain at most _one_ root certificate: all nodes and all clients were obliged to rely on this certificate's authority.
 From version 7.1, Couchbase Server permits multiple root certificates to maintained in a _trust store_ for the cluster.
-This:
-
-* Allows an individual node either to use a CA that is also used by one or more other nodes; or to use a CA that is used by no other node.
+This allows an individual node either to use a CA that is also used by one or more other nodes; or to use a CA that is used by no other node.
 This may be used to increase security: ensuring that a node can be successfully authenticated by some applications, but not by others.
```

Comment:
This may be used during CA certificate rotation I would say.
You upload New CA, then change nodes certificates one by one, then remove old CA.

Reply:
I have changed to:

This may be used during CA certificate rotation: a new CA is uploaded, node certificates are changed one by one, and finally, the old CA is removed.